mkt/detail/tests/test_views.py

```diff
@@ -171,10 +171,11 @@ def test_queries(self, _mock):
 
         The queries we are expecting are:
           * 2 savepoints
+          * 1 tag query for distinguishing homescreen/webapps in indexer task
```

Comment:
why indeed. turns out the indexer call is done in a `post_request_task` which carries over from test setup, and is not done by the code being tested at all.